inspirehep/modules/workflows/workflows/article.py

```diff
@@ -192,6 +193,7 @@
             refextract,
         ]
     ),
+    set_document_type_and_refereed,
```

Comment:
@ammirate not sure if this is the right place for it, ideally it would run after a merge and halt for curation, so it can take into account all info in the record.

Reply:
I think @iulianav put it here to be coherent with the enhancing and, in particular, for `extract_journal_info` have some benefit from it. But yes, have it when the record is completed makes sense.
At this point, I would put it in `POSTENHANCE_RECORD`.